Views/Metas.liquid

```diff
@@ -0,0 +1,29 @@
+
+{% RegisterLink, Href: "/Themes/ShowOrchard.Theme/Images/Favicons/favicon.ico", Rel: "shortcut icon", Type: "image/x-icon" %}
+{% RegisterLink, Href: "/Themes/ShowOrchard.Theme/Images/Favicons/android-chrome-36x36.png", Rel: "image/png" %}
+{% RegisterLink, Href: "/Themes/ShowOrchard.Theme/Images/Favicons/android-chrome-48x48.png", Rel: "image/png" %}
+{% RegisterLink, Href: "/Themes/ShowOrchard.Theme/Images/Favicons/android-chrome-72x72.png", Rel: "image/png" %}
+{% RegisterLink, Href: "/Themes/ShowOrchard.Theme/Images/Favicons/android-chrome-96x96.png", Rel: "image/png" %}
+{% RegisterLink, Href: "/Themes/ShowOrchard.Theme/Images/Favicons/android-chrome-144x144.png", Rel: "image/png" %}
+{% RegisterLink, Href: "/Themes/ShowOrchard.Theme/Images/Favicons/android-chrome-192x192.png", Rel: "image/png" %}
+{% RegisterLink, Href: "/Themes/ShowOrchard.Theme/Images/Favicons/android-chrome-256x256.png", Rel: "image/png" %}
+{% RegisterLink, Href: "/Themes/ShowOrchard.Theme/Images/Favicons/android-chrome-384x384.png", Rel: "image/png" %}
+{% RegisterLink, Href: "/Themes/ShowOrchard.Theme/Images/Favicons/android-chrome-512x512.png", Rel: "image/png" %}
+{% RegisterLink, Href: "/Themes/ShowOrchard.Theme/Images/Favicons/apple-icon.png", Rel: "apple-touch-icon" %}
+{% RegisterLink, Href: "/Themes/ShowOrchard.Theme/Images/Favicons/apple-icon-57x57.png", Rel: "apple-touch-icon" %}
+{% RegisterLink, Href: "/Themes/ShowOrchard.Theme/Images/Favicons/apple-icon-60x60.png", Rel: "apple-touch-icon" %}
+{% RegisterLink, Href: "/Themes/ShowOrchard.Theme/Images/Favicons/apple-icon-72x72.png", Rel: "apple-touch-icon" %}
+{% RegisterLink, Href: "/Themes/ShowOrchard.Theme/Images/Favicons/apple-icon-76x76.png", Rel: "apple-touch-icon" %}
+{% RegisterLink, Href: "/Themes/ShowOrchard.Theme/Images/Favicons/apple-icon-114x114.png", Rel: "apple-touch-icon" %}
+{% RegisterLink, Href: "/Themes/ShowOrchard.Theme/Images/Favicons/apple-icon-120x120.png", Rel: "apple-touch-icon" %}
+{% RegisterLink, Href: "/Themes/ShowOrchard.Theme/Images/Favicons/apple-icon-144x144.png", Rel: "apple-touch-icon" %}
+{% RegisterLink, Href: "/Themes/ShowOrchard.Theme/Images/Favicons/apple-icon-152x152.png", Rel: "apple-touch-icon" %}
+{% RegisterLink, Href: "/Themes/ShowOrchard.Theme/Images/Favicons/apple-icon-180x180.png", Rel: "apple-touch-icon" %}
+
+{% SetMeta, Name: "msapplication-TileColor", Content: "#ffffff" %}
+{% SetMeta, Name: "msapplication-TileImage", Content: "/Themes/ShowOrchard.Theme/Images/Favicons/ms-icon-144x144.png" %}
+{% SetMeta, Name: "msapplication-TileImage", Content: "/Themes/ShowOrchard.Theme/Images/Favicons/ms-icon-150x150.png" %}
+{% SetMeta, Name: "msapplication-TileImage", Content: "/Themes/ShowOrchard.Theme/Images/Favicons/ms-icon-310x310.png" %}
+{% SetMeta, Name: "msapplication-TileImage", Content: "/Themes/ShowOrchard.Theme/Images/Favicons/ms-icon-70x70.png" %}
+{% SetMeta, Name: "theme-color", Content: "#ffffff" %}
```

Comment:
Ez mire van amúgy?

Reply:
Őszintén nem tudom, de minden favicon-os fájlban ott van.

Reply:
Te adtad hozzá, vagy még Robi? De légyszi nézz utána, hogy kell-e.

Reply:
Ah látom, ez az: https://developers.google.com/web/updates/2014/11/Support-for-theme-color-in-Chrome-39-for-Android Jó lesz.